packages/block-library/src/site-tagline/style.scss

```
@@ -0,0 +1,4 @@
+:root :where(.wp-block-site-tagline) {
```

```suggestion
.wp-block-site-tagline {
```

The wrapping `:root :where()` here is unnecessary as both this and a simple class result in `0-1-0` specificity.